src/Nest/Aggregations/AggregationContainer.cs

```diff
@@ -287,6 +290,15 @@ public void Accept(IAggregationVisitor visitor)
 		{
 			if (visitor.Scope == AggregationVisitorScope.Unknown) visitor.Scope = AggregationVisitorScope.Aggregation;
 			new AggregationWalker().Walk(this, visitor);
+		}
+
+		internal static void InjectMetadata(IAggregation aggregation)
+		{
+			if (aggregation.Meta == null)
+				aggregation.Meta = new Dictionary<string, object>();
+			if (aggregation.Meta.ContainsKey(AggregationMetadata.Key))

```

Comment:
Yes...I use to check for that but must have accidentally removed it during refactoring.  Will add it back :+1:.